ui/app/routes/vault/cluster/clients/history.js

```diff
@@ -5,16 +5,19 @@ import { action } from '@ember/object';
 export default class HistoryRoute extends Route {
   async getActivity(start_time) {
```

Comment:
Thought it made sense to _only_ make the request if we had a valid `start_time` from the license, but thoughts on why this could cause problems?